src/python/WMCore/WMSpec/WMWorkload.py

```diff
@@ -689,6 +698,15 @@ def setSiteWhitelist(self, siteWhitelist):
 
         return
 
+    def getSiteBlacklist(self):
+        """
+        Get the site black list from our data
```

Comment:
"Get the site black list for the workflow"

Reply:
ok, done